src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/RuntimeFlavor.cs

```diff
@@ -5,7 +5,8 @@ namespace Microsoft.AspNetCore.Server.IntegrationTesting
 {
     public enum RuntimeFlavor
     {
-        Clr,
-        CoreClr
+        None,
```

Comment:
none?

Reply:
So we can tell if nothing was specified and provide a default.